manifest/armv7l/g/glibc.filelist

```diff
@@ -778,6 +778,7 @@
 /usr/local/lib/libthread_db.so
 /usr/local/lib/libutil.a
 /usr/local/lib/locale/locale-archive
+/usr/local/lib64/crew-preload.so
```

Comment:
This should be `lib` instead of `lib64`.

Reply:
Nope. This is intentionally here as an `aarch64` binary. We should document that better.

Reply:
This is for armv7/aarch64 multiarch machines to handle aarch64 binaries.

Most of our "armv7l" machines are now armv7l/aarch64, and this file supports that case.

Reply:
I think this is going to be a problem for older EOL'd devices.  ONLY aarch64 systems have the dual architecture support.  armv7l supports ONLY 32-bit and should continue to use the `lib` instead of `lib64` directories.  What I think we should start doing is adding aarch64 manifest files with `lib64` but continue to use `lib` for armv7l.

Reply:
`lib64` will sit harmlessly on older devices with just `crew-preload.`  It won't be used at all.